contribute/design/1.x/csi/20190606-csi-volume-provisioning.md

```diff
@@ -503,3 +506,11 @@ NA
 
 - Availability of github.com/openebs/csi repo 
 - Enable integration with Travis as the minimum CI tool
+
+## Missing parameters for feature parity with older implementations:
+
+ - Resource limits for target pods
+ - Target pod affinity
+ - Target pod tolerations
+ - Replica anti-affinity
+ - Preferred replica anti-affinity
```

Comment:
Are the above picked from the defaultConfig of the cStor Volume Create CAST Template?
Are there any more custom tunables like LU Threads, format and mount options etc.

Reply:
Yes @kmova , these have been picked from the defaultConfig of cstor Volume Create CAST template.
LU threads have already been added to cstorVOlumeConfig class in the same oep.
Format and mount options were not there in CAST template, these will be provided by CSI when a mount request is received.